src/btree/bt_vrfy.c

```diff
@@ -833,7 +819,7 @@ __verify_key_hs(
          */
         newer_start_ts = older_start_ts;
     }
-
+    WT_TRET(hs_cursor->close(hs_cursor));
```

Comment:
Shouldn't this go in an err: block so if anything else returns a failure we still close the cursor?

Reply:
Fixed in 5478b37.